hw/display/apple_h12p.c

```diff
@@ -205,11 +221,24 @@ static void h12p_gfx_update(void *opaque)
             error_report("Failed to read framebuffer");
             return;
         }
+        size_t size2 = 0;
+        g_autofree uint8_t *buf2 = NULL;
+        if (s->genpipe1_plane_end && s->genpipe1_plane_start) {
+            size2 = s->genpipe1_plane_end - s->genpipe1_plane_start;
+            buf2 = g_malloc(size2);
+            if (dma_memory_read(&s->dma_as, s->genpipe1_plane_start, buf2, size2, MEMTXATTRS_UNSPECIFIED) != MEMTX_OK) {
+                error_report("Failed to read framebuffer2");
+                return;
+            }
+        }
 
         uint8_t *dest = surface_data(surface);
         for (size_t i = 0; i < s->height; i++) {
-            h12p_draw_row(s, dest, buf + i * stride, s->width, 0);
-            dest += stride;
+            //h12p_draw_row(s, dest, buf + i * stride, s->width, 0);
+            //dest += stride;
```

Comment:
Remove these comments.

Reply:
m'kay.

Reply:
Just didn't wanted to remove your code.

Reply:
It's fine; Git has a change history, lol